build.sbt

```diff
@@ -42,23 +53,50 @@ lazy val scaladocSettings = Seq(
   autoAPIMappings := true
 )
 
-lazy val gitSettings = Seq(
-  git.baseVersion := "0.9",
-  git.gitTagToVersionNumber := {
-    case ReleaseTag(version) => Some(version)
-    case _ => None
+lazy val publishingSettings = Seq(
+  publishTo := {
+    val nexus = "https://oss.sonatype.org/"
+    if (version.value.trim.endsWith("SNAPSHOT"))
+      Some("snapshots" at nexus + "content/repositories/snapshots")
+    else
+      Some("releases" at nexus + "service/local/staging/deploy/maven2")
   },
-  git.formattedShaVersion := {
-    val suffix = git.makeUncommittedSignifierSuffix(git.gitUncommittedChanges.value, git.uncommittedSignifier.value)
-
-    git.gitHeadCommit.value map { _.substring(0, 7) } map { sha =>
-      git.baseVersion.value + "-" + sha + suffix
+  publishMavenStyle := true,
+  pomIncludeRepository := { _ => false },
+  pomExtra := {
+    <url>https://github.com/functional-streams-for-scala/fs2</url>
+    <scm>
+      <url>git@github.com:functional-streams-for-scala/fs2.git</url>
+      <connection>scm:git:git@github.com:functional-streams-for-scala/fs2.git</connection>
+    </scm>
+    <developers>
+      {for ((name, username) <- contributors) yield
+      <developer>
+        <id>{username}</id>
+        <name>{name}</name>
+        <url>http://github.com/{username}</url>
```

Comment:
Good catch -- the tuple extractor is backwards. Should be `for ((username, name) <- ...`